core/src/main/scala/com/nvidia/spark/rapids/tool/qualification/PluginTypeChecker.scala

```diff
@@ -201,14 +225,16 @@ class PluginTypeChecker(platform: Platform = PlatformFactory.createInstance(),
         exprName ++ sqlFuncNames.map(name => (name, cols(1)))
       case "exprs" =>
         // Logic for unsupported expressions
-        if (cols(1) == "NS" && cols(3) != NONE) {
-          val exprName = Seq((cols(0), cols(3)))
+        val opSupVal = OpSuppLevel.fromString(cols(1))
+        if (OpSuppLevel.suppLevelsWithPropagation.contains(opSupVal)) {
+          val loadedReason = opSupVal.getNotesForDisabledOp(cols(3))
```

Comment:
If notes is None, we can use the description of the OpSuppLevel enum

Reply:
This code only uses the description of `OpSuppLevel` enum if the `opSupVal` is `TOFF`, for `NS` and `CO` if the notes is `None`, it wouldn't enter the condition, is that right?

Reply:
Yes, that's on purpose because that's what was the original code doing. It only loads the Reason if it was not "None".
With the new code, the TOFF will be explained in the output of the qualification tool which is is a nice tip when we force disable things on our side.